tests/Unit/Facade.Tests/ConsignmentShipfilesFacadeServiceTests/WhenAdding.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Linn.Stores.Facade.Tests.ConsignmentShipfilesFacadeServiceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+
+    using FluentAssertions;
+
+    using Linn.Common.Facade;
+    using Linn.Stores.Domain.LinnApps.Consignments;
+    using Linn.Stores.Domain.LinnApps.ConsignmentShipfiles;
+    using Linn.Stores.Resources;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenAdding: ContextBase
+    {
+        private IResult<ConsignmentShipfile> result;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.ConsignmentRepository.FindBy(Arg.Any<Expression<Func<Consignment, bool>>>())
+                .Returns(new Consignment {});
+            this.Repository.FindAll().Returns(new List<ConsignmentShipfile> { new ConsignmentShipfile { Id = 1 } }.AsQueryable());
```

Comment:
is this findall used?

Reply:
It is for generating the Id, but you're previous comment to just use a sequence is a better idea anyway